src/main/java/core/basesyntax/strategy/PurchaseOperation.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.Inventory;
+
+public class PurchaseOperation implements FruitOperationHandler {
```

Comment:
The class name `PurchaseOperation` should be more descriptive. According to the checklist, class names should be nouns and might be more specific to reflect their purpose. Consider renaming it to reflect that it handles the purchase operation for fruits, e.g., `FruitPurchaseHandler`.